deezer/client.py

```diff
@@ -90,9 +90,9 @@ def _process_json(self, item, parent=None):
             result[parent.type] = parent
 
         if "type" in result:
-            object_class = self.objects_types.get(result["type"], Resource)
+            object_class = self.objects_types[result["type"]]
         else:
-            object_class = self.objects_types.get(parent, Resource)
+            object_class = self.objects_types[parent]
```

Comment:
Yes, thank you. That turned out to be a bad choice to use `.get(...)` from myself 🎉